src/test/java/kitchenpos/ui/MenuGroupAcceptanceTest.java

```diff
@@ -0,0 +1,38 @@
+package kitchenpos.ui;
+
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import kitchenpos.dto.MenuGroupRequest;
+import kitchenpos.dto.MenuGroupResponse;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.hamcrest.Matchers.hasItems;
+
+public class MenuGroupAcceptanceTest extends AcceptanceTest {
```

Comment:
```suggestion
class MenuGroupAcceptanceTest extends AcceptanceTest {
```
public 을 제거하라고 말해 주네요 ㅎㅎ